videos/videos_model.js

```diff
@@ -0,0 +1,63 @@
+const db = require("../data/db_config");
```

Comment:
Good in keeping the naming conventions solid here, but now I'm gonna hit a merge conflict with your work :)